source/includes/ref-toc-aggregation-pipeline-operator.yaml

```diff
@@ -113,21 +113,26 @@ description: |
    aggregations capable of characterizing data across multiple 
    dimensions, or facets, in a single stage.
 ---
+name: :pipeline:`$bucket`
+file: /reference/operator/aggregation/bucket
+description: |
+   Categorizes incoming documents into groups, called buckets, based on
+   a specified expression and bucket boundaries.
+---
```

Comment:
Alternatives:

Categorizes incoming documents into groups, called buckets, based on a specified expression. Each bucket has a user-specified range of the expression's values.

Categorizes incoming documents into groups, called buckets, based on a specified expression and the boundaries of each bucket.


If one of these sounds better and is used instead, change the first sentence of bucket.txt as well.